modules/serverless-installing-cli-windows.adoc

```diff
@@ -0,0 +1,24 @@
+// Module is included in the following assemblies:
+//
+// serverless/installing-knative-client.adoc
+
+[id="installing-cli-windows_{context}"]
+= Installing the `kn` CLI for Windows
+
+The CLI for Windows is provided as a zip archive. You can download it from the link:TBD[Red Hat Customer Portal].
+
+.Procedure
+
+. Log in to the Red Hat Customer Portal. You must have an active subscription to access the downloads page.
+
+. Download the link:TBD[CLI] from the Red Hat Customer Portal.
```

Comment:
https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest

remove the text 'from the Red Hat Customer portal'